actions/destroy.py

```diff
@@ -21,8 +22,12 @@ def run(self, plan_path, state_file_path, terraform_exec,
         """
         os.chdir(plan_path)
         self.terraform.terraform_bin_path = terraform_exec
-        return_code, stdout, stderr = self.terraform.destroy(plan_path, var_file=variable_files,
-                                                             var=variable_dict,
-                                                             state=state_file_path,
-                                                             force="true")
+        return_code, stdout, stderr = self.terraform.destroy(
+            dir_or_plan=plan_path,
+            var_file=variable_files,
+            var=variable_dict,
+            state=state_file_path,
+            force=IsFlagged,
+            capture_output=False
```

Comment:
why are we not capturing output here?

Reply:
Wanted to match plan, apply, init in that the output is streamed in real-time and kept in their respective stderr/stdout